libyul/optimiser/VariableAssignmentMap.cpp

```diff
@@ -0,0 +1,39 @@
+#include <libyul/optimiser/VariableAssignmentMap.h>
+
+using std::set;
+using namespace solidity::yul;
+
+void VariableAssignmentMap::insert(YulString const& _variable, std::set<YulString> const& _references)
```

Comment:
Shouldn't this be called `set` if it also clears?

Reply:
Sure.